Assets/Puppet/PuppetCrashes.cs

```diff
@@ -146,6 +148,11 @@ public void LastCrashReport()
     }
 
     public static ErrorAttachmentLog[] GetErrorAttachmentstHandler(ErrorReport errorReport)
+    {
+        return GetErrorAttachments();
+    }
+
+    private static ErrorAttachmentLog[] GetErrorAttachments()
     {
         return new ErrorAttachmentLog[]
```

Comment:
Please check UI and return only valid attachments. This always returns two logs which are sent with unexpected data.